File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio15.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio15 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o número de dias trabalhados: ");
        int diasTrabalhados = entrada.nextInt();

        int dia = 8 * 25;
        int salario = diasTrabalhados * dia;
        System.out.println("Salário: R$ " +salario);
    }
}

File: guanabara-c/lista01/exercicio15.c
#include <stdio.h>

int main(){
    int quantidade_dias_trabalhados, salario_dia, salario;

    printf("Insira a quantidade de dias trabalhados: ");
    scanf("%d", &quantidade_dias_trabalhados);

    salario_dia = 8 * 25;
    salario = quantidade_dias_trabalhados * salario_dia;
    printf("Salário: R$ %d", salario);

    return 1;
}
